### ISSUE TITLE
BIOS Offsets Incorrect/Offset Assumptions

### ISSUE THREAD

**Issue by OdinVex** [IndeedMiners/PolarisBiosEditor#1]
1.7.2 (maybe under) has an offset issue with my [XFX RX 580 8GB GTS XXX BIOS](https://www.techpowerup.com/vgabios/199264/199264). Tech Power Up has a dump of my BIOS already. Same hashes.

Vendor and Device ID are wrong.
```
Bootup Message Offset: 122h <-- I notice other BIOS such as ASUS use 120h (0D 0A ...)
Vendor ID: 5249h <-- Should be 1002h
Device ID: 1002h <-- Should be 67DFh
```
I suspect it is because of the BootupMessageOffset being 122h instead of 120h. Did you forget relative offsets? Not a problem because I can work around it at the moment but eh...

**Comment by IndeedMiners** [comment]
Yep, this is common problem that we haven't fixed. It is there from PBE 1.6.

**Comment by Dwokfur** [comment]
I try to parse and display XW 4170 BIOS files extracted from a laptop chimera giant BIOS pile. It displays Vendor ID 5249 and Device ID 1002, whereas it must be 1002 67E0. Many voltage and fan values are also strange - probably due to parsing problems. However it correctly identifies memory timing tables. Are there any plans to make adjustments in case problems like this? I can provide BIOS samples.
